android/src/newarch/com/reactnativecommunity/imageeditor/ImageEditorModule.java

```diff
@@ -0,0 +1,36 @@
+package com.reactnativecommunity.imageeditor;
+
+import android.os.AsyncTask;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.ReactApplicationContext;
+
+public class ImageEditorModule extends NativeRNCImageEditorSpec {
+  private ImageEditorModuleImpl moduleImpl;
+
+  ImageEditorModule(ReactApplicationContext context) {
+    super(context);
+    moduleImpl = new ImageEditorModuleImpl(context);
+  }
+
+  public static final String NAME = ImageEditorModuleImpl.NAME;
+
+  @Override
+  @NonNull
+  public String getName() {
+    return ImageEditorModuleImpl.NAME;
+  }
+
+  @Override
+  public void onCatalystInstanceDestroy() {
```

Comment:
Isn't `onCatalystInstanceDestroy` deprecated/discouraged? If that's the case, then it should be replaced with `invalidate`

Reply:
I will change it in a separete PR 
also we have a some sort of fix: https://github.com/callstack/react-native-image-editor/pull/119/files